public/js/snackbar.js

```diff
@@ -249,17 +241,14 @@ class Snackbar {
      * also creates its reference in snackbars
      */
     createID() {
-        let id = null;
-
-        //goes through all consecutive numbers to find an id
-        let iterator = 0
-        while (id === null) {
-            //checks if the id already exists, otherwise continues to iterate
-            Snackbar.snackbarIDs.includes(iterator) ? iterator++ : id = iterator;
-        }
+        debugger;
+        let id = 0;
+        // goes through all consecutive numbers to find an id
+        for (; id in Snackbar.snackbarIDs; id++); // checks if the id already exists, otherwise continues to iterate
```

Comment:
Could we just do something like this?
```suggestion
        const id = Math.max(...Snackbar.snackbarIDs) + 1;
```

Reply:
Theoretically the IDs could become fragmented as objects are destroyed.

Reply:
Good point; you're right that the `for` loop prevents fragmentation.